docs/src/_plugins/md-syntax-highlighter.js

```diff
@@ -93,15 +77,13 @@ const lineNumberPlugin = (md) => {
         const lineNumbersWrapperCode = `<div class="line-numbers-wrapper" aria-hidden="true">${lineNumbersCode}</div>`;
 
         const finalCode = rawCode
-            .replace(/<\/div>$/, `${lineNumbersWrapperCode}</div>`)
+            .replace(/<\/pre>\n/, `${lineNumbersWrapperCode}</pre>`)
```

Comment:
`markdown-it` will append a `\n` for highlighted result https://github.com/markdown-it/markdown-it/blob/1529ff4944329eb8080a8558dab85164b6c212cd/lib/renderer.js#L59